README.md

```diff
@@ -78,6 +82,20 @@ found in the `src/` directory.
     - stylesheets/
 ```
 
+`require('uswds')` will load all of Draft U.S. Web Design Standard's JavaScript onto the page. The uswds module itself does not export anything.
```

Comment:
Hey! A few comments:

I'd add "the" before "Draft U.S. Web Design Standard's JavaScript." Also, I'd recommend capitalizing "uswds" in the second sentence (because USWDS is an abbreviation of a proper noun).